controllers/idler/idler_controller_test.go

```diff
@@ -323,7 +323,9 @@ func TestEnsureIdling(t *testing.T) {
 		memberoperatortest.AssertThatIdler(t, idler.Name, cl).
 			HasConditions(memberoperatortest.Running(), memberoperatortest.IdlerNotificationCreated())
 		//check the notification is actually created
-		hostCl, _ := reconciler.GetHostCluster()
+		allclustersCl := reconciler.GetHostCluster()
+		hostCl := allclustersCl[0]
```

Comment:
this could work too, right?
```suggestion
		hostCl := reconciler.GetHostCluster()[0]
```